### PR TITLE
Let H2 run queries again.

### PR DESCRIPTION
**Pull request by JeremybellEU**

### CODE REVIEW

**Comment by JWGmeligMeyling** [comment]
It looks so good that I actually used this as base for #140

**Comment by Azkle** [comment]
Have tested it and it works.

**Comment by JWGmeligMeyling** [comment]
Quick fix for #141

File: src/test/java/me/moodcat/database/DatabaseTestModule.java [hunk 1]
```diff
@@ -17,6 +20,7 @@
     @Override
     protected void configure() {
         super.configure();
+        bind(RoomDAO.class).to(H2RoomDAO.class);
```
Comment (GijsWeterings) [inline src/test/java/me/moodcat/database/DatabaseTestModule.java]: You did this as well on creation on App, is this duplicate or does it serve another purpose?

**Comment by GijsWeterings** [comment]
One small question above, other than that LGTM

**Pull request merged**